src/ethereum/osaka/state.py

```diff
@@ -503,9 +509,21 @@ def increase_recipient_balance(recipient: Account) -> None:
 
     modify_state(state, sender_address, reduce_sender_balance)
     modify_state(state, recipient_address, increase_recipient_balance)
-
-
-def set_account_balance(state: State, address: Address, amount: U256) -> None:
+    
+    if change_tracker is not None:
+        sender_new_balance = get_account(state, sender_address).balance
+        recipient_new_balance = get_account(state, recipient_address).balance
+        
+        change_tracker.track_balance_change(sender_address, U256(sender_new_balance), state)
```

Comment:
Here we are expecting methods on the `change_tracker` itself, but the way it's currently implemented, the methods are [standalone methods](https://github.com/ethereum/execution-specs/pull/1338/files#diff-bf1fcde011d1f672973f27cd5a023f0921d07b186d5742e76e8d8e77c04b639eR217) with the tracker as the first argument.

I lean toward this syntax here where the methods are called on the class itself but we should reconcile this one way or the other.